### PR TITLE
feat(HBO Max): cover art

### LINKED ISSUE #5596: HBO Max - Add thumbnail options
### Presence name

HBO Max

### Description

The option to view the thumbnails for shows' and movies' covers in the large image of the presence, rather than just the HBO Max logo. 

### Alternatives

_No response_

### Additional context

An example of this working would be Netflix which does the same, or even YouTube which adds the option for the channel's image or the thumbnail for the video itself (although that doesn't apply here, the way Netflix works would be just fine).

### PR DESCRIPTION
**Pull request by Slowlife01**
**Describe the changes in this pull request:**
Resolves #5596

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

image

</details>

### CODE REVIEW

File: websites/H/HBO Max/presence.ts [hunk 1]
```diff
@@ -44,13 +44,32 @@ const presence = new Presence({
...
+			);
+	});
+}
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
 			largeImageKey: "lg"
 		},
-		video: HTMLVideoElement = document.querySelector("video"),
+		video = document.querySelector("video"),
```
Comment (EncryptedDev) [inline websites/H/HBO Max/presence.ts:72]: Why did you remove the type?
```suggestion
		video = document.querySelector<HTMLVideoElement>("video"),
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/H/HBO Max/presence.ts (hunk 1 above)
Comment (Slowlife01) [reply websites/H/HBO Max/presence.ts:72]: Because it's unnecessary, this query already gave you a type...

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]
